pkgs/development/misc/resholve/default.nix

```
@@ -98,6 +43,8 @@ rec {
   # funcs to validate and phrase invocations of resholve
   # and use those invocations to build packages
   resholve-utils = callPackage ./resholve-utils.nix {
-    inherit resholve;
+    # we can still use resholve-utils without triggering a security warn
+    # this is safe since we will only use `resholve` at build time
+    resholve = removeKnownVulnerabilities resholve;
```

Can't we do something like
```suggestion
    resholve = resholve.override {
       python2 = removeKnownVulnerabilities python2;
     };
```